tests/integrationv2/providers.py

```diff
@@ -15,13 +15,18 @@ class Provider(object):
     ServerMode = "server"
 
     def __init__(self, options: ProviderOptions):
+        # If the provider includes stderr output on a success, set this to True.
+        self.expect_stderr = False
```

Comment:
By default in OpenSSL messages to the users are sent to stderr Ref: https://unix.stackexchange.com/questions/131394/why-does-openssl-print-to-stderr-for-a-successful-command

We should update the OpenSSL(Provider) init function to include `self.expect_stderr = True`.

Reply:
https://github.com/aws/s2n-tls/blob/023d2dd49146b32045eab109871bfd205ca186cb/tests/integrationv2/providers.py#L249-L250